myia/anf_ir.py

```diff
@@ -64,6 +64,22 @@ def output(self, value: 'ANFNode') -> None:
         else:
             self.return_ = Apply([Constant(primops.return_), value], self)
 
+    def add_parameter(self):
+        """Add a new parameter to this graph (appended to the end)."""
+        p = Parameter(self)
+        self.parameters.append(p)
+        return p
+
+    def constant(self, obj):
+        """Create a constant for the given object."""
+        return Constant(obj)
+
+    def apply(self, inputs, graph):
```

Comment:
Why does this take `graph`?

Reply:
That's a mistake from copy pasting the version of that function I had implemented on `Environment`. Should be `def apply(self, *inputs)`.